source/install/troubleshooting.rst

```diff
@@ -60,6 +60,15 @@ Locked out of System Administrator account
 
 If email sign-in was turned off before the System Administrator switched sign-in methods, sign up for a new account and promote it to System Administrator from the command line.
 
+Youtube Videos show a "Video not found" preview
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First, make sure the YouTube video exists by pasting a link to the video on your browser.
+
```

Comment:
Make sure the YouTube video exists by pasting a link to the video into your browser's address bar.